saos-batch/src/test/java/pl/edu/icm/saos/batch/core/indexer/JudgmentReindexingJobTest.java

```diff
@@ -87,7 +84,7 @@ public void judgmentReindexingJob() throws Exception {
         
```

Comment:
Here you check only the number of reindexed judgments. Do you have any test (maybe in another test class) that checks if reindexing correctly changes judgment attributes in solr?